application/models/Files.php

```diff
@@ -139,9 +139,26 @@ public static function addFile(Model_Users $user, Array $fileData, $thumbnail_da
 			return self::PROCESS_OK;
 		}
 		
-		public static function addFolder(Model_Users $user, $name = null, $description = null, Model_Files $parent = null)
+		public static function addFolder(Model_Users $user, $name = null, $description = null, Model_Files $parent = null, $tags = null)
 		{
-			$folder	=	new Model_Files($name, $description, 1, $user, $parent);
+			
+			//We manage the tags
+			$arrayTags	=	explode(' ', $tags);
+			$arrayTagsInstances = [];
+			
+			foreach($arrayTags as $tagName)
+			{
+				$tag = Model_Tags::getTag($tagName);
+				if(!$tag)
+				{
+					$newTag = new Model_Tags($tagName);
+					$tag = Model_Tags::add($newTag);
```

Comment:
VOici une erreur que j'obtiens à l'utilisation de addMultiple() (j'ai vérifié c'est bien cette comande qui fait buger) :
Call to undefined method Model_Tags::reverse() in C:\wamp\www\ShyComics\application\vendors\EntityPHP\Entity.php on line 827